.github/workflows/run-all-wagon-tests.yml

```diff
@@ -0,0 +1,86 @@
+name: 'Run the tests in all wagons with a specific core revision'
+
+on:
+  workflow_dispatch:
+    inputs:
+      core_ref:
+        description: Use a specific version of the core for the workflow run. Defaults to master.
+        default: ""
+      wagon_dependency_ref:
+        description: Use a specific version of the wagon dependency for the workflow run. Defaults to master.
+        default: ""
+      wagon_ref:
+        description: Use a specific version of the wagon dependency for the workflow run. Defaults to master.
+        default: "master"
+  push:
+    branches: [ master ]
+    paths-ignore:
+      - 'doc/**'
+      - '**.md'
```

Comment:
I am unsure about the `push` trigger.

I suggest to at least skip runs where only the VERSION or the CHANGELOG change. 
But: Can we maybe omit the push-trigger completely?

Alternatively, if we keep it, we could remove the schedule in the wagons. If every push to the core triggers all the wagons, the integration-test does not need to be checked on a schedule, but becomes event-driven. Since the remote workflows all notify statuscope, the dashboard continues to work as expected.